Documentation/DataSources/DataSources.md

```diff
@@ -12,6 +12,8 @@ Workbooks support these data sources:
 * [Workload Health](#workload-health)
 * [Azure Resource Health](#azure-resource-health)
 * [Custom Endpoint](#custom-endpoint)
+* [Azure Data Explorer](#azure-data-explorer)
+* [Azure Resource Manager](#azure-resource-manager)
```

Comment:
Should we include custom provider docs?

Reply:
Let's talk about what example we can provide. I can add this as a separate PR